Test/Integration/RebrowseTriggerManagerTests.cs

```diff
@@ -66,76 +75,143 @@ public async Task TestRebrowseIsNotSubscribed(RebrowseTriggersConfig config)
         public async Task TestRebrowseIsTriggered()
         {
             // Arrange
-            var pusher = new DummyPusher(new DummyPusherConfig());
+            var cdfPusher = new DummyPusher(new DummyPusherConfig());
             tester.Config.Extraction.RebrowseTriggers = new RebrowseTriggersConfig
             {
                 Targets = new RebrowseTriggerTargets { NamespacePublicationDate = true }
             };
-            using var extractor = tester.BuildExtractor(true, pushers: pusher);
-
-
-            // Act
+            tester.Config.StateStorage = new StateStorageConfig
+            {
+                Database = StateStoreConfig.StorageType.LiteDb,
+                Location = "lite-state-rebrowse.db",
```

Comment:
We do this in some other tests, delete the state store file in a try block when starting the test.